src/go/util/util.go

```diff
@@ -155,3 +155,8 @@ func MaybeTruncateSpanName(spanName string) string {
 	newSpanName := spanName[:SpanNameMaxByteNum-3] + "..."
 	return newSpanName
 }
+
+var HardCodedSkipServiceControlMethods = []string{
```

Comment:
nit: need to add comment for exported types

// HardCodedSkipServiceControlMethods is a list of methods that ...

https://go.dev/doc/comment